mockturtle/app/scheduled.rb

```diff
@@ -0,0 +1,76 @@
+require "ircinch"
+require "rss"
+require "open-uri"
+require "time"
+require "sqlite3"
+
+class RSSWatcher
+  def initialize(bot, db)
+    sleep 5
+    @bot = bot
+    @db  = db
+  end
+
+  def watch_github
```

Comment:
Need to fix persistence so we can actually start scraping stories.

Should probably fail open and set lastSeen as a class variable. So even if we don't have working persistence we can still scrape rss.

Reply:
Yeah. We should use ActiveRecord here. For all I grumble about it, the consistency of matching the site database access would be worth it.